patches/gcc/6.3.0/950-bionic-android.patch

```diff
@@ -0,0 +1,17618 @@
+Based on https://github.com/crystax/android-toolchain-gcc-6/commits/master
```

Comment:
Correction: rather, libstdc++.a and libstdc++.so are not installed because of this patch. I confirmed that removing 950-bionic-androd.patch fixes libstdc++ installation.

With the patch, x86_64-unknown-linux-uclibc samples:
````
$ find /home/avn/x-tools/x86_64-unknown-linux-uclibc -name "libstdc++*"
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++.*-gdb.py
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++fs.a
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++.*-gdb.py
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++fs.a
````

Without the patch, same sample:
````
$ find /home/avn/x-tools/x86_64-unknown-linux-uclibc -name "libstdc++*"
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++.so.6
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++.a
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++.so
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++.so.6.0.22
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++.so.6.0.22-gdb.py
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/sysroot/lib/libstdc++fs.a
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++.so.6
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++.a
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++.so
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++.so.6.0.22
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++.so.6.0.22-gdb.py
/home/avn/x-tools/x86_64-unknown-linux-uclibc/x86_64-unknown-linux-uclibc/lib64/libstdc++fs.a
````

This is obviously a no-go issue.